radicals/urls.py

```diff
@@ -1,21 +0,0 @@
-"""radicals URL Configuration
```

Comment:
radicals folder should have this urls.py file, your app should probably not even start without it.